packages/api/source/rest/membership.routes.ts

```diff
@@ -0,0 +1,27 @@
+import type { Router, Request, Response, NextFunction } from "express";
+import { constants } from "@hypertool/common";
+
+import { memberships } from "../controllers";
+
+const { httpStatuses } = constants;
+
+const attachRoutes = (router: Router): void => {
+    router.get(
+        "/invitation/accept/:jwt",
+        async (request: Request, response: Response, next: NextFunction) => {
+            const { jwt } = request.params;
+            const result = await memberships.verify(null, jwt);
+
+            if (result)
```

Comment:
Where are the braces?

Reply:
added braces. though it works without braces too